keyboards/neodox/keymaps/neodimio/config.h

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
This file should only contain overrides and not duplicate the keyboard level config

Reply:
done

Reply:
This file can be deleted.

Reply:
This file still needs to be deleted